lib/elixir/lib/code.ex

```diff
@@ -665,11 +665,15 @@ defmodule Code do
       when non-existing atoms are found by the tokenizer.
       Defaults to `false`.
 
+    * `:pairing_metadata` - including metadata about `do/end` blocks,
```

Comment:
For consistency with the options above:

```suggestion
    * `:pairing_metadata` - when `true`, includes metadata about `do/end` blocks,
```